README.md

```diff
@@ -11,165 +12,193 @@ Then add `bs-faker` as a dependency to `bsconfig.json`
 
 # Usage
 
-Don't hesitate to view the interface file `Faker.rei` to see library usage.
+Don't hesitate to view the interface files `*.rei` to see library usage.
```

Comment:
I guess we can remove this section! lots of good usage docs in this README by the contributors 😸